tracer/src/Datadog.Trace.Tools.Runner/RunCiCommand.cs

```diff
@@ -4,43 +4,56 @@
 // </copyright>
 
 using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
 using System.Globalization;
 using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
 using System.Xml;
-using System.Xml.XPath;
 using Datadog.Trace.Agent.DiscoveryService;
 using Datadog.Trace.Ci;
 using Datadog.Trace.Ci.Tags;
 using Datadog.Trace.Logging;
 using Datadog.Trace.Util;
 using Spectre.Console;
-using Spectre.Console.Cli;
 
 namespace Datadog.Trace.Tools.Runner
 {
-    internal class RunCiCommand : AsyncCommand<RunCiSettings>
+    internal class RunCiCommand : CommandWithExamples
     {
         private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(RunCiCommand));
+
         private readonly ApplicationContext _applicationContext;
+        private readonly RunSettings _runSettings;
+        private readonly Option<string> _apiKeyOption = new("--api-key", "Enables agentless with the Api Key");
 
         public RunCiCommand(ApplicationContext applicationContext)
+            : base("run", "Run a command and instrument the tests")
         {
             _applicationContext = applicationContext;
+
+            _runSettings = new(this);
+            AddOption(_apiKeyOption);
+
+            AddExample("dd-trace ci configure azp");
```

Comment:
I think this is the wrong example:

```suggestion
            AddExample("dd-trace ci run -- dotnet test");
```

Reply:
Good catch